sdk/index.ts

```diff
@@ -291,6 +294,29 @@ export class UserKvStorage {
   }
 }
 
+export class PluginEnvStorage {
+  private readonly plugin_id: string;
+
+  public constructor(plugin_id: string) {
+    this.plugin_id = plugin_id;
+  }
+
+  public getString(key: string): string {
+    const request = new HostPluginEnvGetStringRequest(key);
+    const requestBytes = Protobuf.encode<HostPluginEnvGetStringRequest>(
+      request,
+      HostPluginEnvGetStringRequest.encode,
+    );
+    const responsePtr = host.plugin_env_get_string(writeBufferToPr(requestBytes));
+    const response = Protobuf.decode<HostPluginEnvGetStringResponse>(
+      readBufferFromPtr(responsePtr),
+      HostPluginEnvGetStringResponse.decode,
+    );
+    
+    return response.value || "";
```

Comment:
This should probably return `string  | null` instead.

Reply:
Addressed